converters/fromZigbee.js

```diff
@@ -5050,6 +5050,15 @@ const converters = {
             return {humidity: calibrateAndPrecisionRoundOptions(humidity, options, 'humidity')};
         },
     },
+
+    smartthings_acceleration: {
+        cluster: 'manuSpecificSamsungAccelerometer',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            return {acceleration: msg.data['acceleration'] === 1 ? true : false};
```

Comment:
Is this a state or an event? To me it looks like an event if that is indeed the case you can change this to `return {action: 'acceleration'};`

Reply:
To me it's a state. The device sends a message when I pickup the device, and then sends another message after it stops moving for a few seconds.

Reply:
Clear! In that case `accelerating` or `moving` would be a better name

Reply:
Sorry for the delay, been away.
I've updated to use "moving" as suggested